Compile error with quick-xml 0.33.0

I get the following compile error with the latest quick-xml:
```
error[E0308]: mismatched types
    --> src/xml_output.rs:145:31
     |
145  |         let event = Event::PI(BytesText::new(concat.as_str()));
     |                     --------- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `BytesPI<'_>`, found `BytesText<'_>`
     |                     |
     |                     arguments to this enum variant are incorrect
     |
note: tuple variant defined here
```
@hitenkoku Could you look into this?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.8.9 - 2024-07-02]
+
+Fixed a compiler error when using the latest `quick-xml`. (@YamatoSecurity)
+
 ## [0.8.8 - 2024-02-16]
 
 Removed unused crates `serde` and `simplelog`.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 ## [0.8.9 - 2024-07-02]
 
-Fixed a compiler error when using the latest `quick-xml`. (@YamatoSecurity)
+Fixed a compiler error when using the latest `quick-xml`. (@hitenkoku)
 
 ## [0.8.8 - 2024-02-16]
 

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -6,7 +6,7 @@ repository = "https://github.com/Yamato-Security/hayabusa-evtx"
 license = "MIT"
 readme = "README.md"
 
-version = "0.8.8"
+version = "0.8.9"
 authors = ["Omer Ben-Amram <omerbenamram@gmail.com>, Yamato Security"]
 edition = "2021"
 
@@ -18,7 +18,7 @@ crc32fast = "1.*"
 chrono = { version = "0.4.*", features = ["serde"] }
 encoding = "0.*"
 byteorder = "1"
-quick-xml = "0.*"
+quick-xml = "*"
 thiserror = "1"
 log = { version = "0.*", features = ["release_max_level_debug"] }
 winstructs = "0.*"

--- a/src/json_output.rs
+++ b/src/json_output.rs
@@ -475,7 +475,7 @@ mod tests {
     /// Converts an XML string to JSON, panics in xml is invalid.
     fn xml_to_json(xml: &str, settings: &ParserSettings) -> String {
         let mut reader = Reader::from_str(xml);
-        reader.trim_text(true);
+        reader.config_mut().trim_text(true);
 
         let mut output = JsonOutput::new(settings);
         output.visit_start_of_stream().expect("Start of stream");

--- a/src/xml_output.rs
+++ b/src/xml_output.rs
@@ -7,7 +7,7 @@ use log::trace;
 use std::io::Write;
 
 use quick_xml::events::attributes::Attribute;
-use quick_xml::events::{BytesDecl, BytesEnd, BytesStart, BytesText, Event};
+use quick_xml::events::{BytesDecl, BytesEnd, BytesStart, BytesText, Event, BytesPI};
 use quick_xml::Writer;
 
 use crate::binxml::name::BinXmlName;
@@ -142,7 +142,7 @@ impl<W: Write> BinXmlOutput for XmlOutput<W> {
         // PITARGET - Emit the text "<?", the text (as specified by the Name rule in 2.2.12), and then the space character " ".
         // Emit the text (as specified by the NullTerminatedUnicodeString rule in 2.2.12), and then the text "?>".
         let concat = pi.name.as_str().to_owned() + pi.data.as_ref(); // only `String` supports concatenation.
-        let event = Event::PI(BytesText::new(concat.as_str()));
+        let event = Event::PI(BytesPI::new(concat.as_str()));
         self.writer.write_event(event)?;
 
         Ok(())

--- a/src/xml_output.rs
+++ b/src/xml_output.rs
@@ -7,7 +7,7 @@ use log::trace;
 use std::io::Write;
 
 use quick_xml::events::attributes::Attribute;
-use quick_xml::events::{BytesDecl, BytesEnd, BytesStart, BytesText, Event, BytesPI};
+use quick_xml::events::{BytesDecl, BytesEnd, BytesPI, BytesStart, BytesText, Event};
 use quick_xml::Writer;
 
 use crate::binxml::name::BinXmlName;

